apps/openchallenges/challenge-service/src/main/resources/db/challenges.csv

```diff
@@ -520,4 +520,36 @@
+"526","tcr-specificity-prediction-challenge","IMMREP23: TCR Specificity Prediction Challenge","Competitors will make predictions on previously unpublished TCR-epitope bind...","IMMREP23, the second annual IMMREP benchmark on TCR-epitope specificity prediction will run from November 1, 2023 to December 11, 2023. Together with several experimental groups, we have compiled a data set of paired TCR data with annotated specificity to 21 pHLA (covering 6 distinct HLA molecules).  This challenge models TCR epitope recognition as a binary classification task. For a given test set of TCR-epitope pairs, the task of the model is to identify which pairs will bind and which will not bind.","","https://www.kaggle.com/competitions/tcr-specificity-prediction-challenge","completed","8","","2023-10-31","2023-12-11","2242","","2024-11-20 17:17:44"
+"527","","","","","","","","","","\N","\N","\N","",""
+"528","","","","","","","","","","\N","\N","\N","",""
+"529","","","","","","","","","","\N","\N","\N","",""
+"530","","","","","","","","","","\N","\N","\N","",""
+"531","","","","","","","","","","\N","\N","\N","",""
+"532","","","","","","","","","","\N","\N","\N","",""
+"533","","","","","","","","","","\N","\N","\N","",""
+"534","","","","","","","","","","\N","\N","\N","",""
+"535","","","","","","","","","","\N","\N","\N","2024-11-19 22:20:48","2024-11-19 22:20:48"
+"536","","","","","","","","","","\N","\N","\N","",""
+"537","","","","","","","","","","\N","\N","\N","",""
+"538","","","","","","","","","","\N","\N","\N","",""
+"539","","","","","","","","","","\N","\N","\N","",""
+"540","","","","","","","","","","\N","\N","\N","",""
+"541","","","","","","","","","","\N","\N","\N","",""
+"542","","","","","","","","","","\N","\N","\N","",""
+"543","","","","","","","","","","\N","\N","\N","",""
+"544","","","","","","","","","","\N","\N","\N","",""
+"545","","","","","","","","","","\N","\N","\N","",""
+"546","","","","","","","","","","\N","\N","\N","",""
+"547","","","","","","","","","","\N","\N","\N","2024-11-19 22:20:52","2024-11-19 22:20:52"
+"548","","","","","","","","","","\N","\N","\N","",""
+"549","","","","","","","","","","\N","\N","\N","",""
+"550","","","","","","","","","","\N","\N","\N","",""
+"551","","","","","","","","","","\N","\N","\N","",""
+"552","","","","","","","","","","\N","\N","\N","",""
+"553","","","","","","","","","","\N","\N","\N","",""
+"554","","","","","","","","","","\N","\N","\N","",""
```

Comment:
@mdsage1 The spreadsheet must be updated to remove these lines.

Reply:
@tschaffter These lines were added by @gaiaandreoletti since I don't have permissions to change the 'id' column. I will ask her to remove them.